gulpfile.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 const gulp = require('gulp');
-const boilerplate = require('appium-gulp-plugins').boilerplate.use(gulp);
+const boilerplate = require('@appium/gulp-plugins').boilerplate.use(gulp);
```

Comment:
The package won't work after publish if we don't add `index.js` explicitly to boilerplate files. See for example https://github.com/appium/appium-uiautomator2-server/pull/455/files#diff-25789e3ba4c2adf4a68996260eb693a441b4a834c38b76167a120f0b51b969f7R14

Reply:
I see. As https://github.com/appium/node-simctl/pull/153/commits/7bf2efa5c4f3a444e88063acbf153576021690c6 ?
This repository does not have a top-level index.js, so this repo needs it.

Reply:
you are right, this is not needed for this repository: "main": "./build/lib/simctl.js",